example_app_generator/generate_action_mailer_specs.rb

```diff
@@ -31,6 +31,7 @@
   if skip_active_record?
     comment_lines 'spec/support/default_preview_path', /active_record/
     comment_lines 'spec/support/default_preview_path', /active_storage/
+    comment_lines 'spec/support/default_preview_path', /action_mailbox/
```

Comment:
This is the key change to skip loading AR in `no_ar_example_app`, it's patching `default_preview_path`.